lib/cocoapods-rome/post_install.rb

```diff
@@ -31,17 +30,47 @@ def build_for_iosish_platform(sandbox, build_dir, target, device, simulator)
   end
 end
 
-def xcodebuild(sandbox, target, sdk='macosx', deployment_target=nil)
-  args = %W(-project #{sandbox.project_path.realdirpath} -scheme #{target} -configuration #{CONFIGURATION} -sdk #{sdk})
+def xcodebuild(sandbox, target, sdk='macosx', deployment_target=nil, configuration)
+  args = %W(-project #{sandbox.project_path.realdirpath} -scheme #{target} -configuration #{configuration} -sdk #{sdk})
   platform = PLATFORMS[sdk]
   args += Fourflusher::SimControl.new.destination(:oldest, platform, deployment_target) unless platform.nil?
   Pod::Executable.execute_command 'xcodebuild', args, true
 end
 
-Pod::HooksManager.register('cocoapods-rome', :post_install) do |installer_context|
+def enable_debug_information(project_path, configuration)
+  project = Xcodeproj::Project.open(project_path)
+  project.targets.each do |target|
+    config = target.build_configurations.find { |config| config.name.eql? configuration }
+    config.build_settings['DEBUG_INFORMATION_FORMAT'] = 'dwarf-with-dsym'
+    config.build_settings['ONLY_ACTIVE_ARCH'] = 'NO'
+  end
+  project.save
+end
+
+def copy_dsym_files(dsym_destination, configuration)
+  dsym_destination.rmtree if dsym_destination.directory?
+  platforms = ['iphoneos', 'iphonesimulator']
+  platforms.each do |platform|
+    dsym = Pathname.glob("build/#{configuration}-#{platform}/**/*.dSYM")
+    dsym.each do |dsym|
+      destination = dsym_destination + platform
+      FileUtils.mkdir_p destination
+      FileUtils.cp_r dsym, destination, :remove_destination => true
+    end
+  end
+end
+
+Pod::HooksManager.register('cocoapods-rome', :post_install) do |installer_context, user_options|
+  enable_dsym = user_options['dsym']
+  enable_dsym ||= true
```

Comment:
this will make it impossible to disable, let's use `enable_dsym = user_options.fetch('dsym', true)`

Reply:
Ah, nice to know that about conditional assignment.